packages/ui-components/src/components/range/range.tsx

```diff
@@ -0,0 +1,92 @@
+import { Component, Host, h, Prop, Element, EventEmitter, Event, State, Watch } from '@stencil/core';
+import { IRange, IRangeEvents } from './range.types';
+
+@Component({
+	tag: 'kv-range',
+	styleUrl: 'range.scss',
+	shadow: true
+})
+export class KvRange implements IRange, IRangeEvents {
```

Comment:
Please add the component to the angular module in `kv-ui-components.module` file